magento/adobe-stock-integration
# See more link is visible even if there are less than 4 related images

### Description (*)

This issue was found whilst investigating https://github.com/magento/adobe-stock-integration/issues/514. See more link is visible for the merchant/user if there are less than 4 related images for the record.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Search the keyword with text`'298323255`
5. Press enter
6. Click on first image to expand image preview
7. Click on more from the series tab in related image section
8. Check for the see more link

### Actual result
There is a see more link for the record

### Expected result
There should not be see more link for the record because there is only three related series image
